.github/workflows/pre-release.yml

```diff
@@ -0,0 +1,57 @@
+name: Pre release
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_tag:
+        description: 'Image tag in the format x.x.x'
+        required: true
+        type: string
+
+env:
+  RELEASE_TAG: ${{ inputs.release_tag }}
+  PRERELEASE_DOCS_BRANCH: 'dg8d45z'
+
+jobs:
+  ## TODO we can add a condition like github.actor.role == 'Maintainer' to limit trigger to maintainers only
```

Comment:
should we do this as part of this PR itself? I think we should.

Reply:
This is not yet supported, but there's active discussion ongoing. When it's implemented, we can do that.